### ISSUE TITLE
Villages are white boxes.

### ISSUE THREAD

**Issue by awkwaRdcaRRot** [ckhgame/VillageBox#2]
Great mod! Everything is working the way its supposed to so far, except my villagers are just a white box.

**Comment by ckhgame** [comment]
Thank you awkwaRdcaRRot! Could you make a screenshot for that while box?

**Comment by awkwaRdcaRRot** [comment]
I will next time I hop on, but to maybe help in the meantime the box is seemingly if you were to put a box around the actual model and make it white. So same size in height and width and depth as the model. Just a perfect rectangular box and solid white.

**Comment by ckhgame** [comment]
OK I think I got it. I'm not sure what causes that problem because it never happens on my computer. It sounds like the bounding box are rendering for some reason. Let me check my code. Thank you!
By that way, does that happen very often?

**Comment by awkwaRdcaRRot** [comment]
Happened literally from my first villager and hasn't gone away at all. I
have a lot of other mods on the server, that may be contributing? Next time
I get on I'll run a test with just your mod and see if it happens.

**Comment by ckhgame** [comment]
Yes, it may well be that Village Box is incompatible with some mods you installed.

**Comment by awkwaRdcaRRot** [comment]
Here is the screenshot. I can't test to see if I run it without the mods if it'll work because Curse isn't letting me install or uninstall mods. At the moment. Tonight I will do it the old fashion way and get back to you.

![2016-03-26_14 58 28](https://cloud.githubusercontent.com/assets/14096186/14061907/7034ee1a-f363-11e5-911d-92a29b6a4269.png)

**Comment by ckhgame** [comment]
ok, thank you!
